Packs/Kafka/Integrations/KafkaV3/KafkaV3.yml

```diff
@@ -26,7 +26,7 @@ configuration:
   required: false
 - display: Client certificate (.cer)
   name: client_cert
-  type: 12
+  type: 14
```

Comment:
This is ok. You want to change: `client_cert_key`.

Reply:
[Fixed](https://github.com/demisto/content/pull/29683/commits/004f3d8f22c31d5c213b18f5ea127ee9f5aa14d7)